how_to_add_new_hotspot.md

```diff
@@ -161,6 +217,123 @@ Sentry API endpoint for `diagnostics` container.
+
+## Modifying settings.ini
+The new hotspot could need to define another I2C bus path. Then it needed to be added into the settings.ini file and `gen_docker_compose.py` file needs to be updated, if necessary.
+
+## Defining configuration parameters in the Balena Fleet
+A single board computer generally wouldn't have a BIOS due to constraints. Therefore manufacturers like Raspberry Pi Foundation created a clever way to keep boot parameters on the device. For example there is a `config.txt` file under `/boot` for Rasspberry Pi SBC's and it defines the parameters which is cruical to make the device work properly.
+
+In Balena, such critical parameters are defined in Fleet Configuration and passed down to the device. So before adding a new device type into a new fleet configuration parameters need to be set in Fleet Configuration.
+
+## Defining variant and frequency in the Belana Fleet
+There are two critical parameters which has to be defined in a Balena Fleet before adding a new device into it.
+
+### VARIANT
+This parameter defines which variant will be used in the whole firmware stack. The name is the key from the `variant_definitions` dictionary which is detailed above.
+
+**Example:**
+| Name | Value | Service
+|------|-------|--------
+| VARIANT | NEBHNT-IN1 | All Services
+
+### FREQ
+Like in many things, the world also has been divided about the RF frequency spectrum. So fleets are separated also with their center frequencies because hotspots acts differently regarding to their RF bands.
+
+If this parameter wouldn't be defined in a fleet, the `packet_forwarder` container could not progress further and waits for this parameters in a restart loop.
+
+The expected values are: 470, 868 and 915.
+
+**Example:**
+| Name | Value | Service
+|------|-------|--------
+| FREQ | 868 | All Services
+
+### HELIUM_MINER_HEIGHT_URL
+This parameter is added to test fleets. It is an API endpoint used for fetching current miner height from Stakejoy API. Current value is [this link](https://fuzzy-marmalade-warlock.skittles.stakejoy.com/v1/blocks/height)
+
+### OVERRIDE_CONFIG_URL
+This parameter is added to test fleets. It helps testing new miner configurations. Current value is [this link](https://helium-assets-stage.nebra.com/docker.config)
+
+### Sentry Entries
+We use Sentry.io for monitoring the performance and status of our hotspots. Every critical container needs a Sentry API endpoint address.
+
+#### SENTRY_CONFIG
+Sentry API endpoint for `gateway_config` container.
+
+#### SENTRY_DIAG
+Sentry API endpoint for `diagnostics` container.
+
+#### SENTRY_PKTFWD
+Sentry API endpoint for `packet_forwarder` container.
+
+
 ## Modifying hotspot-production-images repo
 This repo is used to create production images for the devices. The scripts in the repo could be used with automation pipeline or manually. The YAML files under the `.github/workflows` folder are the CI/CD automation pipeline configurations.
```

Comment:
a lot of this file seems to be duplicated for some reason. dont really understand how/why

Reply:
I think if we just revert it to this one - https://github.com/NebraLtd/helium-miner-software/commit/7541882dab18e1c3db4c4659f2e2922ac875755a

Then it will be correct

Doesn't look like anything added since then?